core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -66,8 +67,8 @@ static void generate1(ClusterGenerator clusterGenerator) {
         serverProperties.put("foo", "bar");
         clusterGenerator.accept(ClusterConfig.defaultBuilder()
                 .setTypes(Collections.singleton(Type.ZK))
-                .setName("Generated Test")
                 .setServerProperties(serverProperties)
+                .setTags(Arrays.asList("name", "Generated Test"))
```

Comment:
` .setTags(Collections.singletonList("Generated Test"))`

Reply:
thanks for the comment!
`singletonList` is a more flexible wrapper of a list, that's the reason why it's better than just using Arrays, right?